spec/datadog/core/remote/client_spec.rb

```diff
@@ -6,6 +6,11 @@
 require 'datadog/core/remote/client'
 
 RSpec.describe Datadog::Core::Remote::Client do
+  before(:all) do
```

Comment:
This will impact all tests, not only in this file, if it's possible I would prefer to do just `before(:each)`